yarn.lock

```diff
@@ -9125,6 +9007,21 @@ execa@^4.0.0, execa@^4.0.2:
     signal-exit "^3.0.2"
     strip-final-newline "^2.0.0"
 
+execa@^4.0.2:
```

Comment:
Can you please try removing both of these `execa@^4.0.0` and `execa@^4.0.2`, and do a bootstrap to fix this section.